docs/source/configuration.rst

```diff
@@ -68,11 +81,39 @@ Following is a partial list of most predominant settings specific to `Weaver`.
   | The *path* variant **SHOULD** start with ``/`` for appropriate concatenation with ``weaver.url``, although this is
     not strictly enforced.
 
+- | ``weaver.wps_output_s3_bucket = <s3-bucket>``
+  | (default: ``None``)
+  |
+  | AWS S3 bucket where to store WPS outputs. Used in conjunction with ``weaver.wps_output_s3_region``.
+  |
+  | When this parameter is defined, any job result generated by a process execution will be stored (uploaded)
+    to that location. If no bucket is specified, the outputs fall back to using the location specified by
+    ``weaver.wps_output_dir``.
```

Comment:
And what happens if `weaver.wps_output_s3_bucket` is defined but not reachable?
Fallback to `weaver.wps_output_dir` or thows an exception?

Reply:
It will throw an exception. 
Same as if `weaver.wps_output_dir` points to a directory which the app doesn't have enough permissions to write. 
External setup to weaver provided to it is expected to be configured appropriately, as Weaver cannot do anything about it to fix it anyway.